### PR TITLE
Add logback notice

### PR DESCRIPTION
**Pull request by wsargent**

### CODE REVIEW

File: public/markdown/vulnerabilities/20170407-LogbackDeser.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Impact
+
+This vulnerability is filed as [CVE-2017-5929](https://nvd.nist.gov/vuln/detail/CVE-2017-5929).  As with most Java deserialization bugs, deserializing untrusted input leads to remote command execution in the JVM.
+
+Because Play is not configured out of the box using SocketAppender, and because SocketAppender does not fit into production logging environments (i.e. syslog or ELK stack), it is unlikely that production environments are impacted by this vulnerability.  There is a possibility of development environments being configured explicitly using SocketAppender to integrate into IDEs or desktop logging tools.
+
+## Affected Versions
+
+Play 2.0.0 - 2.5.14
```
Comment (gmethvin) [inline public/markdown/vulnerabilities/20170407-LogbackDeser.md]: oops, this should say 2.5.13

File: public/markdown/vulnerabilities/20170407-LogbackDeser.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Workarounds
+
+The workaround for this vulnerability is to upgrade the Logback library to version 1.2.3, which does not have these issues.
+
+// https://mvnrepository.com/artifact/ch.qos.logback/logback-classic
+libraryDependencies += "ch.qos.logback" % "logback-classic" % "1.2.3"
+
+## Fixes
+
+Upgrade to Play 2.5.14, upgrade the Logback library if you are on previous versions of Play, and remove references to SocketAppender or SocketReceiver from Logback configuration.
```
Comment (gmethvin) [inline public/markdown/vulnerabilities/20170407-LogbackDeser.md]: s/and/or?

**Comment by wsargent** [comment]
ok updated

**Comment by mkurz** [comment]
This wasn't sent out to the Play Framework Security mailing list
https://groups.google.com/forum/?fromgroups#!forum/play-framework-security

**Comment by wsargent** [comment]
@mkurz good catch, fixed.

**Pull request merged**